sum-of-nth-terms/sum-of-nth-terms.py

```diff
@@ -0,0 +1,5 @@
+def series_sum(n):
+    total = 0
+    for i in range(n):
+        total += 1/(1+(i*3))
```

Comment:
This integer division will only return "1" in Python 2. You should make all your code Python 2/3 compatible.